code/renderers/react/template/stories/docgen-components/ts-function-component/input.tsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 
 import { imported } from '../imported';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore (css import not supported in TS)
+import styles from '../imported.module.css';
```

Comment:
This was causing this issue in the repro.

I had to add the ts-ignore, and couldn't make it a expect-error.
This errors when running the `check` task, but it does not when loaded into storybook, so a expect-error would actually cause issues there.

Reply:
See:
https://app.circleci.com/pipelines/github/storybookjs/storybook/71811/workflows/33e08595-1907-487c-b454-418ca1f3eeae/jobs/632550